ml_nlp/models/sentiment-analysis-nlp-accuracy-83.ipynb

```diff
@@ -0,0 +1,1290 @@
+{
```

Comment:
@Cephasgokula it is working fine. However we'll suggest you some edits in placement of created files:

1. Move the notebook (.ipynb) from `model` to `code` directory.
2. Using your jupyter notebook, save the final model (you may use pickle) and place it in `model` directory.

---

Try to convert both train and test data in uniform format (either in Excel or CVS only).